pkgs/applications/version-management/git-lfs/default.nix

```
@@ -2,23 +2,26 @@
 
 buildGoModule rec {
   pname = "git-lfs";
-  version = "3.4.1";
+  # Inclues git compatibility fixes which are in master, but no official release
+  version = "unstable-2023-01-10";
```

```suggestion
  version = "3.4.1-unstable-2023-01-10";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming